spark/src/main/resources/python/zeppelin_pyspark.py

```diff
@@ -98,6 +98,15 @@ def checkbox(self, name, options, defaultChecked = None):
 
     checkedIterables = self.z.checkbox(name, defaultCheckedIterables, optionIterables)
     return gateway.jvm.scala.collection.JavaConversions.asJavaCollection(checkedIterables)
+  
+  def registerCallback(self, event, cmd, replName="pyspark"):
```

Comment:
@felixcheung The space is omitted in the optional argument in order to comply with PEP8, see [here](https://www.python.org/dev/peps/pep-0008/#other-recommendations). 

The main purpose of defaulting this argument to "pyspark" is solely for convenience, ie so you could easily call `registerCallback()` without specifying the REPL name manually every time it is called.